target/linux/rtl838x/files-5.4/drivers/net/ethernet/rtl838x_eth.h

```diff
@@ -0,0 +1,263 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#ifndef _RTL838X_ETH_H
+#define _RTL838X_ETH_H
+
+// #define RTL838X_SW_BASE ((int )0xBB000000)
```

Comment:
From checkpatch.pl:
`ERROR: do not use C99 // comments`

Is this needed anyway?

Same comment for line 84.

Reply:
Will be removed.